apps/web/src/app/[locale]/tracks/_components/track-challenge-card.tsx

```diff
@@ -131,7 +131,7 @@ export function MockTrackChallenge({ challenge }: { challenge: Challenge }) {
           <Check className="absolute left-1 my-auto h-3 w-3 scale-0 stroke-[4] text-white duration-300 peer-checked:scale-100 dark:text-black" />
           {challenge.name}
         </div>
-        <div className="hidden md:block">
+        <div className="md:block">
```

Comment:
I don't know if this is necessary, isn't it block by default? if we're removing hidden on mobile I don't think `md:block` is needed
```suggestion
        <div>
```